pipeline/src/client.py

```diff
@@ -259,6 +263,20 @@ def reprocess(self, start_date=None, end_date=None):
     def create_all_views(self):
         return self.flagged.create_views_all_flags()
 
+    ###########################################################
+
+    def init_flag_dict(self):
+        self.flag_lookup = dict()
+        df = self.flags.get_full_table()
+
+        if df is None:
+            return
+
+        FlagInfo = namedtuple("FlagInfo", "id desc")
+
+        for row in df.itertuples():
+            self.flag_lookup[row.name] = FlagInfo(row.flag_id, row.description)
+
```

Comment:
This could fail if the user fails to adjust the flags table after changing the codebase; my first comment about this would resolve this issue though.

Reply:
Since this method returns nothing an isn't going to be in the constructor, this makes it seem like it cannot not fail, but it can fail. Could you address this by doing something like returning a boolean or recreating the flags table if it's invalid?